include/rangers-api/ApplicationCommon/Reflection/CaptionAttribute.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+namespace app_cmn::rfl {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.